queries/perl/highlights.scm

```diff
@@ -0,0 +1,169 @@
+; Variables
+(variable_declaration
+  .
+  (scope) @keyword)
+[
+(single_var_declaration)
+(scalar_variable)
+(array_variable)
+(hash_variable)
+(hash_variable)
+] @variable
+
+
+[
+(package_name)
+(special_variable)
+(special_literal)
+(super)
+] @constant
+
+(
+  [
+  (package_name)
+  (super)
+  ]
+  .
+  ("::" @constant)
```

Comment:
Constant usually  means an identifier, isn't this more a `@punctuation.delimiter/@punctuation.special/@operator`

Reply:
This is a hack to highlight `::` operator to joining a package name & a function.

* before
   - <img width="429" alt="スクリーンショット 0003-08-30 12 08 09" src="https://user-images.githubusercontent.com/1239245/131279654-fe3f9b21-7e77-4d5d-b188-07097cf66f97.png">
* after
   - <img width="540" alt="スクリーンショット 0003-08-30 12 08 22" src="https://user-images.githubusercontent.com/1239245/131279669-64ad4331-11d1-4b5f-9092-43895c111ec1.png">

I'm using `@contant` because the prefix package name is `@constant`. But as you say, it is not suitable for this, maybe.

Reply:
then this should be highlighted as an operator, we have the same for cpp https://github.com/nvim-treesitter/nvim-treesitter/blob/e31bba5d97bfbc59d94a47f40e7ad9e36a5a53ee/queries/cpp/highlights.scm#L149-L149